packages/tooltip/src/components/popover/Popover.tsx

```diff
@@ -15,26 +15,20 @@ export type PopoverVariant =
   | "error"
   | "outlined";
 
+export type LightMode = "light" | "dark";
 export interface PopoverProps
   extends Partial<Omit<TippyComponentProps, "theme" | "render">> {
   tippyRef?: TippyCallbackRef<HTMLDivElement>;
   disablePadding?: boolean;
   lazy?: boolean;
   variant?: PopoverVariant;
+  lightMode?: LightMode;
```

Comment:
I think all tooltips should be dark. No light mode.

Reply:
Oh, and this is a Popover, that shouldn't be dark.

Reply:
Tooltip uses the popover component in the render:
![image](https://user-images.githubusercontent.com/6305945/204849143-57f1cc1e-77d7-40ed-855d-3f77d78e992f.png)

Hence why I'm defaulting to light in the Popover component, but adding an option to let it be dark.